kratos/containers/variables_list.h

```diff
@@ -150,7 +154,11 @@ namespace Kratos
 				std::equal(mVariables.begin(), mVariables.end(), r.mVariables.begin());
 		}
 
-
+		//public API of intrusive_ptr
+		unsigned int use_count() const noexcept
```

Comment:
Just for me to know, why do we want the `use_count()` to not throw exceptions?

Reply:
It affects the performance and we know that we are not sending exception there.

Reply:
also it comes in the recipe ;-)